### ISSUE TITLE
FF asks for fingerprint to access passwords on iPhone that does not have touch-ID capability

### ISSUE THREAD

**Issue by InfiniteCookie** [mozilla-mobile/firefox-ios#12403]
## Steps to reproduce

attempting to access passwords in FF on iPhone results in 'enter your passcode or touch ID to access passwords' message.  This phone does not have a touchID home button--only biometric ID allowed is faceID

### Expected behavior

App should offer 'enter your passcode or FaceID to access passwords' and offer a button for FaceID.

### Actual behavior

I can only see my passwords by entering my Passcode.

### Device & build information
* Device: iPhone 12 mini, iOS 16.0.2
* Build version: FF for iOS 106.2 (21661)
### Notes
Attachments:

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-5272)

**Comment by dnarcese** [comment]
duplicate of #12094

**Comment by InfiniteCookie** [comment]
Thank you.  I see that now and would happily cancel this one if I knew how.

I do have a better sequence to contribute regarding how this happened, however:  
Since I posted that first request, I deleted the app and re-download it. When I first clicked to go to the password area, I got a request to allow face ID to be used. If I clicked past that request, then I would see the touch ID, or enter code option.  I think someone clicking too quickly, as I was, may miss that first button and end up in the endless loop of impossible request for touch ID.  It would be great if the app simply included someway to request that setting be reset if it goes wrong.
